requestor-tutorials/flash-tutorial-of-requestor-development/README.md

```diff
@@ -201,7 +201,7 @@ yagna payment status
 
 If, after a few minutes, you still can't see the tokens, re-run the `yagna payment fund` command above and check again after a few more minutes.
 
-As the last resort, if you suspect that there is a more serious issue with the zkSync payment driver or our faucet, you may wish to completely do away with using it and fall back to the older, on-chain payment driver. In such case, please refer to instructions in [our troubleshooting section](../../troubleshooting/common-issues.md#payment-driver-initialization-issue).
+As the last resort, if you suspect that there is a more serious issue with the payment driver or our faucet, you may wish to completely do away with using it and fall back to the older, on-chain payment driver. In such case, please refer to instructions in [our troubleshooting section](../../troubleshooting/common-issues.md#payment-driver-initializ§ation-issue).
```

Comment:
Is the link really pointing to "../../troubleshooting/common-issues.md#payment-driver-initializ§ation-issue" (with the `§` character)?

Reply:
indeed, not, good catch, @stranger80 <3